third_party/move/move-prover/tests/sources/functional/bitwise_operators.exp

```diff
@@ -0,0 +1,3 @@
+Move prover returns: [internal] boogie exited with compilation errors:
+bitwise_operators.bpl(3532,23): Error: invalid type for argument 0 in application of $IsEqual'u64': bv64 (expected: int)
```

Comment:
I'm afraid we cannot merge with this unresolved. This is effecting the production prover. There must be some change applied to the move model which leaks out of the v2 compiler, because this code path is not depending on the v2 compiler at all, only on the model.

Reply:
I made a mistake trying to avoid code duplication between the existing functions `ExpData::free_vars` and `ExpData::free_local_vars_with_node_id` and my needed method `ExpData::get_free_local_vars`.  I failed to notice that `free_local_vars_with_node_id` actually had little to do with free variables, so my re-implementation was broken.  I've restored the original implementation and renamed it as `bound_local_vars_with_node_id`, which is more accurate.

Since I've now gone to the effort to rename that one, I'm going to rename the `free_vars` function as `free_vars_with_types` so I can use that name for my new method that just returns the `free_vars`.

Reply:
So this is now fixed.

Reply:
Can we close #10944 then?

Reply:
Done.